packages/api/src/hooks/usePOIStatus.ts

```diff
@@ -0,0 +1,14 @@
+import useGetAccountStatus from './useGetAccountStatus';
+
+const usePOIStatus = () => {
+    const { data: get_account_status_data, ...rest } = useGetAccountStatus();
+
+    const poi_status = get_account_status_data?.authentication?.identity?.status;
+
+    return {
+        has_submitted_poi: poi_status === 'none',
```

Comment:
```ts
        /** This represents the current status for proof of identity document submitted for authentication. */
        data: status,
        /** Determine if the user has submitted a proof of identity document. */
        is_submitted: status === 'none',
        /** Blah blah blah */
        is_expired: status === 'expired',
        /** Blah blah blah */
        is_pending: status === 'pending',
        /** Blah blah blah */
        is_rejected: status === 'rejected',
        /** Blah blah blah */
        is_suspected: status === 'suspected',
        /** Blah blah blah */
        is_verified: status === 'verified',
```
@yashim-deriv Could we do something like this? To provide more data and better documentation 🤔